### PR TITLE
docs(contributing): install and use yarn 1

### PR DESCRIPTION
**Pull request by PixnBits**
## **Description**
Currently `$ npm install yarn` installs v2

## **Motivation** 
this project uses yarn v1 for installing

## **Test** **Conditions**
none

#### Check boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation (adding or updating documentation)
- [ ] Dependency update
- [ ] Security update

#### Check boxes that apply:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] I have added the Apache 2.0 license header to any new files created.

### CODE REVIEW

**Review by 10xLaCroixDrinker — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ This project adheres to the American Express [Code of Conduct](./CODE_OF_CONDUCT
 3. Install the dependencies by running
 
     ```bash
-    $ yarn
+    $ yarn@^1
```
Comment (PixnBits) [inline CONTRIBUTING.md:36]: wait, this was supposed to be the install command 🤦

**Review by 10xLaCroixDrinker — CHANGES_REQUESTED** [review]
https://github.com/americanexpress/one-app-cli/pull/504/files/3ed27ab5f6f372146af2cf9024a757bd8f653a2d#r1134373356

File: CONTRIBUTING.md (hunk 1 above)
Comment (smackfu) [reply CONTRIBUTING.md:36]: Maybe just add a line saying to use yarn@1?  Reinstalling it globally might not be appreciated.

Comment (PixnBits) [reply CONTRIBUTING.md]: changed to `$ npx yarn@^1` in 89e092c

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by JAdshead — APPROVED** [review]

**Pull request merged**